api/facadeversions.go

```diff
@@ -36,7 +36,6 @@ var facadeVersions = map[string]int{
 	"CAASOperatorProvisioner":      1,
 	"CAASOperatorUpgrader":         1,
 	"CAASUnitProvisioner":          2,
-	"CharmHub":                     1,
```

Comment:
?

Reply:
This is only used to allow the cli to make a BestFacadeVersion("charmhub") check. There's no need for it if we check resources instead. There's no client api that otherwise uses this facade.